### PR TITLE
CC Libraries Adapter calls

### PR DESCRIPTION
**Pull request by volfied**
- Adapter functions to save text and layer style thumbnails and export to file
- Adapter functions to apply layer style and text styles to active layers

Hoping these are the last adapter calls we need. @iwehrman Can you review?

### CODE REVIEW

File: src/lib/libraries.js [hunk 1]
```diff
@@ -196,9 +196,36 @@ define(function (require, exports) {
             });
     };
 
+    /**
+     * Creates a thumbnail preview for the character style
+     *
+     * @param {string} path Location to save the thumbnail to
+     * @param {string} postscriptName Typeface postscript name
+     * @param {string} text Preview text to be written in thumbnail
+     * @param {number} size Size of the preview text
```
Comment (iwehrman) [inline src/lib/libraries.js]: Is there any more information you can provide here about the `size` parameter? Is this like point or pixel size of the text? Or is this the side length in pixels of a square thumbnail?

File: src/lib/textLayer.js [hunk 2]
```diff
@@ -570,6 +570,76 @@ define(function (require, exports) {
...
+     *
+     * @param {ActionDescriptor} sourceRef
+     * @param {object} style Primary representation of the type style
+     *
+     * @return {PlayObject}
+     */
+    var applyTextStyle = function (sourceRef, style) {
+        assert(referenceOf(sourceRef) === "textLayer", "applyTextStyle expects a textLayer reference");
+        
+        sourceRef = {
```
Comment (iwehrman) [inline src/lib/textLayer.js]: Hmm, this is weird. Does this mean that there is only one valid `sourceRef`? If that's the case, why even have the parameter?

**Comment by iwehrman** [comment]
Done with a first pass. Back to you, @volfied.

File: src/lib/textLayer.js (hunk 2 above)
Comment (volfied) [inline src/lib/textLayer.js]: I should have added the same warning comment other functions in this file has. Seems like textLayer reference resolving doesn't work and it has to be the active layer.

**Comment by volfied** [comment]
Thanks @iwehrman , clarified the jsdoc and the reference issue.

**Pull request merged**